\documentclass[]{article}

\usepackage{graphicx}
\usepackage{enumerate}
\usepackage[version=4,arrows=pgf]{mhchem}
\usepackage{chemfig}
\usepackage{amsmath}

%opening
\title{structural bioinformatics - assignment 4}
\author{Johanna Becher, Talha Rehman}
\date{}

\begin{document}

\maketitle

\section{Theory exercises}
\subsection{Task 1}
Mutations are changes in a cells genetic information. They can be classified by their formation or by their impact they have. \newline
They can form by insertion or deletion or the change of a single base in the DNA. But it can also happen that a sequence of bases is inserted or deleted.\newline
If the mutation only consists of one base there are diffrent classifications of the mutation depending on the amino acid that is build with the triplet containing the mutation.\newline
A silent mutation occurs when the change of the base doesn't affect the generation of the amino acid at all. A missense mutation leads to a change of the amino acid and a nonsense mutation causes the build in of an early stop codon. \newline

\subsection{Task 2}
Bacteria multiply quite fast by copying it's DNA and dividing into 2 new cells. During the copying mechanism there is a risk that errors occur which would cause a mutation. \newline
Those mutations can have a lethal impact or they can have even no impact at all. But if the mutation causes an adaption of the bacteria to a given condition, due to Darwin's theory of natural selection, it is very propable that this bacteria is going to grow better.\newline
If a bacteria culture is exposed to an antibiotic it could happen, that somme bacteria mutate and their descendants become resistant to this antibiotic.

\subsection{Task 3}
When it comes to modelling of 3 dimensional protein structure we want to be able to compare different models and how "good" they are. Therefore we compare our models to experimantally resolved  protein structures by using a measure for structure similarity.\newpage 

\noindent One of those measures is called RMSD, which calculates the root mean square deviation of the \ce{C\alpha}-atoms of n superimposed amino acids. \newline
\begin{displaymath}
	RMSD(v,w)=\sqrt{\frac{1}{n}\sum_{i=1}^{n}(||v_i-w_i||^2)}
\end{displaymath}
Where $v = (v_1,..,v_n)$ is the first protein with n amino acids, w the second protein and $vi = (v_{ix}, v_{iy}, v_{iz})$ consists of the coordinates of the i-th \ce{C\alpha}-atom in v (respectively w). \newline
Since RMSD weights all atoms equally it is very sensitive to local structure deviations, but it does not take into account the length of the alignment. This means the shorter the alignment is, the better the RMSD will be. \newline
\newline
Another measure, called GDT\_TS (=global distance test, total score) calculates the largest set of \ce{C\alpha}-atoms of two superimposed protein structures. It calculates a percentage of how many \ce{C\alpha}-atoms have a distance smaller than a predefinded cutoff. \newline
It is more accurate than the RMSD and is used as a major assessment criteria in the CASP.\newline

\subsection{Task 4 and 5}
We were looking at the protein \textit{TNF-alpha} which is secreted by activated macrophages. It causes the necrosis of tumors (what the name already indicates) but it also plays a role in different signaling pathways.\newline
The top 5 missense mutations are the following:\newline
\includegraphics[height=15ex, width=450pt]{mutations.png}
\newpage
\noindent In the following picture you see the positions of those 5 mutations marked in red, while the rest of the protein is shown in green.\newline
\includegraphics[height=40ex, width=200pt]{mutation_3d_small.png}
\newline
\newline
The numbers indicate the mutations by their allele frequency,
\begin{enumerate}[1)]
	\item position 84: Pro $\rightarrow$ Leu
	\item position 194: Ile $\rightarrow$ Asn
	\item position 161: Val $\rightarrow$ Ile
	\item position 54: Gly $\rightarrow$ Glu
	\item position 34: Ser $\rightarrow$ Thr
\end{enumerate}

\noindent Since mutations 2) and 5) are insinde a secondary structure they could break up this structure, what could cause a loss of functionality. \newline
Changing Isoleucine into Aspargine could cause some changes. Both amino acids have a similar size but different side chains. \newline
Replacing Serine with Threonine would only add a \ce{CH3} group what wouldn't have a big impact on the \ce{\alpha}-helix.\newline
The other 3 muatations are placed in loops connectiong secondaary structures. The first mutation looks quite interesting, because it is quite close to an helix. When you replace Proline with Leucine you get an amino acids which prefers helices more than loops. It could cause an elongation of the \ce{\alpha}-helix. \newline
The same could happen at mutation 4) because Glutamic Acid also likes helices.\newline
\newline
Those changes of the secondary strcture were not detectable with SwissModel, neither for mutation 4) nor mutation 1). 
\end{document}
